ex2/main.py

```diff
@@ -0,0 +1,48 @@
+import string
+import random
+import pandas as pd
+
+from flask import Flask
+
+app = Flask(__name__)
+
+@app.route("/generate_password")
+def generate_password():
+    """
+    from 10 to 20 chars
+    upper and lower case
+    """
+    # string
+    # ascii_lowercase
+    # ascii_uppercase
+    # int
+    # special symbols
+    # return password
+    lower = string.ascii_lowercase
+    upper = string.ascii_uppercase
+    digits = string.digits
+    special_chars = string.punctuation
+    password = ''.join(random.choice(lower) + random.choice(upper) + random.choice(digits) + random.choice(special_chars) for _ in range(random.randint(10, 20)))
```

Comment:
This code is not working correctlly 
![image](https://github.com/user-attachments/assets/8977e5cc-acb8-4c90-b928-97971d03cab2)